### PR TITLE
Topic/improve err messages on tag resource

### PR DESCRIPTION
**Pull request by henrybarreto**

### CODE REVIEW

File: api/services/errors.go [hunk 1]
```diff
@@ -1,32 +1,85 @@
...
-	ErrReport                    = errors.New("report error")
-	ErrMaxTagReached             = errors.New("maximum number of tags reached")
-	ErrDeviceNotFound            = errors.New("device not found")
-	ErrDuplicateTagName          = errors.New("this tag Name already exists")
-	ErrTagNameNotFound           = errors.New("tag Name not found")
-	ErrPublicKeyNotFound         = errors.New("public key not found")
-	ErrPublicKeyInvalid          = errors.New("public key invalid")
-	ErrNoTags                    = errors.New("no tags has found")
-	ErrTypeAssertion             = errors.New("type assertion failed")
+	ErrReport                    = errold.New("report error")
```
Comment (otavio) [inline api/services/errors.go:62]: old package.

File: api/routes/handlers/converter/converter.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package converter
+
+import (
+	"net/http"
+
+	"github.com/shellhub-io/shellhub/api/services"
+)
+
+func FromServiceToHTTP(code int) int {
```
Comment (otavio) [inline api/routes/handlers/converter/converter.go:9]: from what?

File: api/routes/tags.go [hunk 3]
```diff
@@ -61,14 +55,6 @@ func (h *Handler) RenameTag(c gateway.Context) error {
 		switch err {
 		case guard.ErrForbidden:
 			return c.NoContent(http.StatusForbidden)
```
Comment (otavio) [inline api/routes/tags.go:57]: Couldn't the forbidden error be handled generically as well?

**Review by otavio — APPROVED** [review]

**Review by otavio — APPROVED** [review]

**Comment by henrybarreto** [comment]
> Could the constants become private?

Yes, it does, but not now. I still need the constants while the error package transition.

**Pull request merged**